lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -69,7 +69,8 @@ export class AccountEntity {
             homeAccountId: this.homeAccountId,
             environment: this.environment,
             tenantId: this.realm,
-            username: this.username
+            username: this.username,
+            localAccountId: this.localAccountId
```

Comment:
Might this result in cache misses?

Reply:
For apps that upgrade to this version

Reply:
localAccountId is not part of the cacheKey, it is only required as part of the interface.